### PR TITLE
Backend coverage

### PR DESCRIPTION
**Pull request by MalavikaKrishnan400**
added coverage

### CODE REVIEW

File: src/LCT.APICommunications.UTest/SW360ApicommunicationUTest.cs [hunk 1]
```diff
@@ -148,12 +152,191 @@ public void SW360Apicommunication_UpdateComponent_ReturnsInvalidOperationExcepti
...
+                    "SendAsync",
+                    ItExpr.IsAny<HttpRequestMessage>(),
+                    ItExpr.IsAny<CancellationToken>())
+                .ReturnsAsync(new HttpResponseMessage
+                {
+                    StatusCode = HttpStatusCode.OK,
+                });
+
+            var httpClient = new HttpClient(mockHttpMessageHandler.Object);
+            var sW360Apicommunication = new SW360Apicommunication(connectionSettings); // Assuming your class SW360Apicommunication does not have a constructor that accepts HttpClient
```
Comment (karthika-g) [inline src/LCT.APICommunications.UTest/SW360ApicommunicationUTest.cs:203]: remove this comment

File: src/LCT.APICommunications.UTest/SW360ApicommunicationUTest.cs [hunk 2]
```diff
@@ -148,12 +152,191 @@ public void SW360Apicommunication_UpdateComponent_ReturnsInvalidOperationExcepti
...
```
Comment (karthika-g) [inline src/LCT.APICommunications.UTest/SW360ApicommunicationUTest.cs:230]: remove this comment

File: src/LCT.APICommunications.UTest/SW360ApicommunicationUTest.cs [hunk 3]
```diff
@@ -148,12 +152,191 @@ public void SW360Apicommunication_UpdateComponent_ReturnsInvalidOperationExcepti
...
+
+
+
+            var createComponentContent = new CreateComponent(); // Assuming CreateComponent is a class. Replace with your actual CreateComponent instance.
```
Comment (karthika-g) [inline src/LCT.APICommunications.UTest/SW360ApicommunicationUTest.cs:234]: remove this comment

File: src/LCT.APICommunications.UTest/SW360ApicommunicationUTest.cs [hunk 4]
```diff
@@ -148,12 +152,191 @@ public void SW360Apicommunication_UpdateComponent_ReturnsInvalidOperationExcepti
...
+                    ItExpr.IsAny<HttpRequestMessage>(),
+                    ItExpr.IsAny<CancellationToken>())
+                .ReturnsAsync(new HttpResponseMessage
+                {
+                    StatusCode = HttpStatusCode.OK,
+                    Content = new StringContent("Test content"),
+                });
+
+            var httpClient = new HttpClient(mockHttpMessageHandler.Object);
+            var sW360Apicommunication = new SW360Apicommunication(connectionSettings); // Assuming your class SW360Apicommunication does not have a constructor that accepts HttpClient
```
Comment (karthika-g) [inline src/LCT.APICommunications.UTest/SW360ApicommunicationUTest.cs:258]: check and remove all such unwanted comments

File: src/LCT.Services.UTest/JFrogServiceUTest.cs [hunk 5]
```diff
@@ -1,234 +1 @@
...
-            // Act
-            IJFrogService jFrogService = new JFrogService(mockJfrogApiComFacade.Object);
-            AqlResult actual = await jFrogService.GetPackageInfo("energy-dev-npm-egll", "saap-api-node-2.26.3-LicenseClearing.16.sha-058fada.tgz", string.Empty);
-
-            // Assert
-            Assert.Null(actual);
-        }
-    }
-}
+﻿
```
Comment (karthika-g) [inline src/LCT.Services.UTest/JFrogServiceUTest.cs:1]: where did this file content go?

**Review by karthika-g — APPROVED** [review]

File: src/UnitTestUtilities/appsettingsUnitTest.json
Comment (Vijayalakshmi027) [inline src/UnitTestUtilities/appsettingsUnitTest.json:1]: do not check in  this url. undo the change

**Comment by Vijayalakshmi027** [comment]
obsolete
